code/modules/mob/living/carbon/breathe.dm

```diff
@@ -69,11 +69,12 @@
 		return
 
 	for(var/obj/effect/effect/smoke/chem/smoke in view(1, src))
-		if(smoke.reagents.total_volume)
-			smoke.reagents.trans_to_mob(src, 5, CHEM_INGEST, copy = 1)
-			smoke.reagents.trans_to_mob(src, 5, CHEM_BLOOD, copy = 1)
-			// I dunno, maybe the reagents enter the blood stream through the lungs?
-			break // If they breathe in the nasty stuff once, no need to continue checking
+		if(smoke)
```

Comment:
smoke would never be null

Reply:
In fact, I made this change because while testing, a runtime appeared and this fixed it.

Reply:
Null checks don't fix errors. You're just masking it. What would cause the smoke to be null in this case? Do you have the stack log?

Reply:
Would it have been "Can not check null.total volume"? That seems more likely the case

Also, save it for another PR, it's an unrelated change

Reply:
> Can not check null.total volume
yes. 

I know the policy of putting unrelated things in other prs but, it was something I did to make my code work. Also, I already added it to the changelog.

Reply:
>Do you have the stack log?
no

>Null checks don't fix errors. You're just masking it. 


I will remove this then.